### ISSUE TITLE
sqlalchemy can't find dialects:db2.ibm_db

### ISSUE THREAD

**Issue by damache** [ibmdb/python-ibmdbsa#23]
I don't have sudo access where I am installing sqlalchemy.  here is how I installed:
![screenshot from 2017-09-18 16-40-08](https://user-images.githubusercontent.com/11265811/30566042-0d8993ce-9c90-11e7-9b0f-fe2a781f8555.png)

this is the method I'm using 
```
import pandas as pd
from sqlalchemy import create_engine

def load_esx_data():
    sql = "my_sql_stmt"
    con = create_engine("db2+ibm_db://dashdb-entry-yp-dal09-09.services.dal.bluemix.net:50000/BLUDB")
    return pd.read_sql_query(sql,con)
```

and this is the error

```

NoSuchModuleErrorTraceback (most recent call last)
<ipython-input-31-d4e40433e6a4> in <module>()
----> 1 esx_data = load_esx_data()
      2 esx.head()

<ipython-input-30-d5ba26ec9fe0> in load_esx_data()
      4 def load_esx_data():
      5     sql = "select * from DASH7228.ESXCAPACITY UNION select * from DASH7228.ESXCAPACITY1X UNION select * from DASH7228.ESXCAPACITY2X select * from DASH7228.ESXDRCAPACITY select * from DASH7228.ESXDRCAPACITY1X"
----> 6     con = create_engine("db2+ibm_db://dashdb-entry-yp-dal09-09.services.dal.bluemix.net:50000/BLUDB")
      7     return pd.read_sql_query(sql,con)

/gpfs/fs01/user/s5d6-da75800f90bd2c-09e06b09ef9b/.local/lib/python2.7/site-packages/sqlalchemy/engine/__init__.pyc in create_engine(*args, **kwargs)
    389     strategy = kwargs.pop('strategy', default_strategy)
    390     strategy = strategies.strategies[strategy]
--> 391     return strategy.create(*args, **kwargs)
    392 
    393 

/gpfs/fs01/user/s5d6-da75800f90bd2c-09e06b09ef9b/.local/lib/python2.7/site-packages/sqlalchemy/engine/strategies.pyc in create(self, name_or_url, **kwargs)
     54         u.query.pop('plugin', None)
     55 
---> 56         entrypoint = u._get_entrypoint()
     57         dialect_cls = entrypoint.get_dialect_cls(u)
     58 

/gpfs/fs01/user/s5d6-da75800f90bd2c-09e06b09ef9b/.local/lib/python2.7/site-packages/sqlalchemy/engine/url.pyc in _get_entrypoint(self)
    137         else:
    138             name = self.drivername.replace('+', '.')
--> 139         cls = registry.load(name)
    140         # check for legacy dialects that
    141         # would return a module with 'dialect' as the

/gpfs/fs01/user/s5d6-da75800f90bd2c-09e06b09ef9b/.local/lib/python2.7/site-packages/sqlalchemy/util/langhelpers.pyc in load(self, name)
    216         raise exc.NoSuchModuleError(
    217             "Can't load plugin: %s:%s" %
--> 218             (self.group, name))
    219 
    220     def register(self, name, modulepath, objname):

NoSuchModuleError: Can't load plugin: sqlalchemy.dialects:db2.ibm_db

​``

**Comment by hemlatabhatt** [comment]
@damache ,I am not able to reproduce the issue. Can you please check which python its using while running the test case and also can you share your pip list.

Thanks
Hemlata

**Comment by damache** [comment]
@hemlatabhatt I'm using Python 2.7.11
here is the pip list:
colorlover (0.2.1)
cufflinks (0.8.2)
ibm-db-sa (0.3.3)
SQLAlchemy (1.1.14)
wget (3.2)

**Comment by hemlatabhatt** [comment]
@damache  Can you please install ibm_db using command 'pip install ibm_db' and re-try the test case.

**Comment by damache** [comment]
that did it.  I'm able to connect.  thanks for your help.

**Comment by PeterFogh** [comment]
I had the same problem and fixed it with `pip install ibm_db==2.0.8a`.
Why is the dependency between `ibm_db_sa` and `ibm_db` not specified in the installation of `ibm_db_sa`, as a pip package? 

Note, I had to install ibm_db version 2.0.8a, as seen here: https://github.com/ibmdb/python-ibmdb/issues/303

**Comment by hemlatabhatt** [comment]
@PeterFogh  ,i have added the fix in git ,Once released in git ,pip install should be working fine.
Thanks
